packages/web/src/shared/menu-items.ts

```diff
@@ -71,6 +71,10 @@ export const pagePaths = {
     name: 'White Papers',
     link: '/papers',
   },
+  PRESS: {
+    name: 'Presss',
```

Comment:
It looks like this is where that extra 's' is coming from in the other files

Reply:
ah good call.

Reply:
all fixed